Agents/FenlandTrajectoryAgent/README.md

```diff
@@ -1,16 +1,16 @@
 # 1. Description
 
-The `Fenland Trajectory Agent` is a specialized tool for batch instantiation of structured time-series GPS trajectory data. This agent is capable of receiving HTTP POST requests or CURL commands to load GPS trajectory files, which it subsequently instantiates into triples before uploading them into knowledge graphs.
+The `Fenland Trajectory Agent` is a specialized tool for batch instantiation of structured time-series GPS trajectory data. This agent is capable of receiving HTTP POST requests or CURL commands to load GPS trajectory files, which it subsequently instantiates into triples before uploading them into the knowledge graphss.
```

Comment:
Can you remove the two extra "s"s? It should read "the knowledge graph". That was kind of the key point in the previous change request...

Reply:
I'm very sorry for the oversight. I had multiple windows open and committed the wrong change previously. I've corrected it and please refer to commit [1a9507925790a42d6258ead3a2db7cb32a965c8f]

Reply:
For the record, this has now been addressed in commit c209f9d1490df8cc9bb8360f6dc67a77ed640f9c.